lib/draft/block.ex

```diff
@@ -1,74 +1,78 @@
 defmodule Draft.Block do
-  @moduledoc """
-  Converts a single DraftJS block to html.
-  """
+  defmacro __using__(_) do
+    quote do
+      @moduledoc """
+      Converts a single DraftJS block to html.
+      """
 
-  alias Draft.Ranges
+      use Draft.Ranges
 
-  @doc """
-  Renders the given DraftJS input as html.
+      @doc """
+      Renders the given DraftJS input as html.
 
-  ## Examples
-      iex> entity_map = %{}
-      iex> block = %{"key" => "1", "text" => "Hello", "type" => "unstyled",
-      ...>   "depth" => 0,  "inlineStyleRanges" => [], "entityRanges" => [],
-      ...>   "data" => %{}}
-      iex> Draft.Block.to_html block, entity_map
-      "<p>Hello</p>"
-  """
-  def to_html(block, entity_map) do
-    process_block(block, entity_map)
-  end
+      ## Examples
+          iex> entity_map = %{}
+          iex> block = %{"key" => "1", "text" => "Hello", "type" => "unstyled",
+          ...>   "depth" => 0,  "inlineStyleRanges" => [], "entityRanges" => [],
+          ...>   "data" => %{}}
+          iex> Draft.Block.to_html block, entity_map
+          "<p>Hello</p>"
+      """
+      def block_to_html(block, entity_map) do
+        process_block(block, entity_map)
+      end
 
-  defp process_block(%{"type" => "unstyled",
-                      "text" => "",
-                      "key" => _,
-                      "data" => _,
-                      "depth" => _,
-                      "entityRanges" => _,
-                      "inlineStyleRanges" => _}, _) do
-    "<br>"
-  end
+      def process_block(%{"type" => "unstyled",
```

Comment:
I think this answers my question when I was trying to write this myself before. When you write a `use` macro I needed to make these methods public. Thanks for doing this. Reading it and seeing it makes more sense! I like it a lot

Reply:
oops not npm lol but you know what I mean